gradle.properties

```diff
@@ -67,6 +67,8 @@ compileJVM=
 compileJVMVer=8
 
 #JAVA_HOME to be used by tests
+#testJVM=/Library/Java/JavaVirtualMachines/java11/Contents/Home
```

Comment:
Not needed for the PR, but maybe nice as a breadcrumb for future use? Either way, not worth re-running tests just to remove a comment. If there are change requests for Java things, please clean this up, too.

Reply:
Oops! Thanks for catching that.

Reply:
Fixed